docs/dialogflow-events.rst

```diff
@@ -48,3 +48,69 @@ The ``FacebookEvent`` Object
   });
 
   const reply = await facebookBot.execute(event);
+
+  The ``facebookEvent`` object also gives you the necessary helpers to implement the Handover Protocol, very useful when you want to pass the conversation from your bot to a live person, the most common example is when user sends to your bot the following text: I want to talk to a representative. This means your bot is not understanding what user is saying or the bot can't give to the user what they are looking for. So, it's necessary a person to talk directly to the user. You can pass the control to your Page Inbox like this:
+
+.. code-block:: javascript
+
+  const { FacebookEvent, FacebookPlatform, VoxaApp } = require('voxa');
+
+  const config = {
+    pageAccessToken: 'EAAaKuJF183EBAApxv.........',
+  };
+  const app = new VoxaApp({ views, variables });
+  const facebookBot = new FacebookPlatform(app, config);
+
+  app.onIntent("PassControlIntent", async (voxaEvent: FacebookEvent) => {
+    await voxaEvent.facebook.passThreadControlToPageInbox();
+
+    return {
+      flow: "terminate",
+      text: "Facebook.RepresentativeWillGetInTouch.text",
+      to: "die",
+    };
+  });
+
+  Also, if the app you are working on is not the Primary Receiver, you can request control of the conversation like this:
```

Comment:
There seems to be some formatting issue in the rst doc, probably one extra tabe missing or something like that. Check how this is rendered in https://github.com/VoxaAI/voxa/blob/bb0daac7c95ccefd49ca5f1186a0726d25d4b28f/docs/dialogflow-events.rst